release_notes/ocp-4-8-release-notes.adoc

```diff
@@ -70,9 +70,43 @@ This release adds improvements related to the following components and concepts.
 === Registry
 
 
-[id="ocp-4-8-operators"]
+[id="ocp-4-8-olm"]
 === Operator lifecycle
 
+[id="ocp-4-7-admin-error-reporting"]
+==== Enhanced error reporting for administrators
+
+A cluster administrator using Operator Lifecycle Manager (OLM) to install an Operator might encounter several error conditions that are related either to the current API or low-level APIs. Previously, there was little insight into why OLM could not fulfill a request to install or update an Operator. These errors could range from trivial issues like typos in object properties or missing RBAC, to more complex issues where items could not be loaded from the catalog due to metadata parsing.
+
+Because administrators should not require understanding of the interaction process between the various low-level APIs or access to the OLM pod logs to successfully debug such issues, {product-title} 4.8 introduces the following enhancements in OLM to provide administrators with more comprehensible error reporting and messages:
+
+[id="ocp-4-7-installplan-errors"]
+Retrying install plans::
+Install plans, defined by an `InstallPlan` object, can encounter transient errors, for example, due to API server availability or conflicts with other writers. Previously, these errors would result in the termination of partially-applied install plans that required manual cleanup. With this enhancement, the Catalog Operator now retries errors during install plan execution for up to one minute. The new `.status.message` field provides a human-readable indication when retries are occurring.
+
+[id="ocp-4-7-operatorgroup-errors"]
+Indicating invalid Operator groups::
+Creating a subscription in a namespace with no Operator groups or multiple Operator groups would previously result in a stalled Operator installation with an install plan that stays in `phase=Installing` forever. With this enhancement, the install plan immediately transitions to `phase=Failed` so that the administrator can correct the invalid Operator group and try again.
```

Comment:
@adellape "and try again" should be re-creating sub after correct invalid Operator group. if only correcting invalid operator group, not deleting sub and re-creating it will not work.

I am fear that someone understand "try again" wrongly, so better to "try to re-create subscription again." or some else words.